### PR TITLE
feat: Create user-service

### PR DESCRIPTION
**Pull request by shaowi**

### CODE REVIEW

File: backend/.env [hunk 1]
```diff
@@ -0,0 +1,10 @@
+PGHOST=localhost
+PGPORT=5000
+PGUSER=postgres
+PGPASSWORD=123
+PGDATABASE=dev
+PGSSLMODE=disable
+
+EmailFrom= webbforumm@gmail.com
+EmailFromSMTPPw=kdlavcphfzutdgbv
```
Comment (chuakid) [inline backend/.env:9]: We should consider .gitignore-ing .env files. Maybe put a .env.example or something

File: backend/database/connection.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+func Connect() {
+	connection, err := gorm.Open(postgres.New(postgres.Config{
+		DSN: config.GetPostgresConnectionStr(),
+	}), &gorm.Config{})
+
+	if err != nil {
+		panic("could not connect to the database")
+	}
+	DB = connection
+	ClearTables(connection)
```
Comment (chuakid) [inline backend/database/connection.go:24]: Is this deleting all the users on every connect?

File: backend/.env (hunk 1 above)
Comment (shaowi) [reply backend/.env:9]: Okay will add it to gitignore

File: backend/database/connection.go (hunk 2 above)
Comment (shaowi) [reply backend/database/connection.go:24]: yup I was using this code for testing, gonna remove it now

**Review by chuakid — APPROVED** [review]
LGTM!

**Pull request merged**